carla/recourse_methods/processing/counterfactuals.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+import numpy as np
+import pandas as pd
+
+from carla.models.api import MLModel
+
+
+def counterfactual_to_dataframe(
+    mlmodel: MLModel, counterfactuals: List
+) -> pd.DataFrame:
```

Comment:
Nice to factor this out. A function docstring would be nice there to describe the steps.

Reply:
Right, makes totally sense, I'll add this.